src/client.rs

```diff
@@ -55,6 +55,24 @@ pub enum ApiError {
     Other(u16),
 }
 
+/// Generic optional parameters for methods
+pub struct OptionalParams<'a> {
+    pub marker: Option<&'a u32>,
+    pub sort: Option<&'a str>,
+    pub limit: Option<&'a u8>,
+}
+
+/// Optional parameters for an order.
+/// For more info see: <https://bitso.com/api_info?python#place-an-order>
```

Comment:
I think we could remove the "python" from the url. 
"https://bitso.com/api_info?place-an-order#place-an-order"

Reply:
Ah! Thank you!